core/node/node_framework/src/implementations/layers/housekeeper.rs

```diff
@@ -0,0 +1,316 @@
+use std::time::Duration;
+
+use zksync_config::configs::{
+    fri_prover_group::FriProverGroupConfig, house_keeper::HouseKeeperConfig,
+    FriProofCompressorConfig, FriProverConfig, FriWitnessGeneratorConfig,
+};
+use zksync_core::house_keeper::{
+    blocks_state_reporter::L1BatchMetricsReporter,
+    fri_proof_compressor_job_retry_manager::FriProofCompressorJobRetryManager,
+    fri_proof_compressor_queue_monitor::FriProofCompressorStatsReporter,
+    fri_prover_job_retry_manager::FriProverJobRetryManager,
+    fri_prover_queue_monitor::FriProverStatsReporter,
+    fri_scheduler_circuit_queuer::SchedulerCircuitQueuer,
+    fri_witness_generator_jobs_retry_manager::FriWitnessGeneratorJobRetryManager,
+    fri_witness_generator_queue_monitor::FriWitnessGeneratorStatsReporter,
+    periodic_job::PeriodicJob,
+    waiting_to_queued_fri_witness_job_mover::WaitingToQueuedFriWitnessJobMover,
+};
+use zksync_dal::ConnectionPool;
+
+use crate::{
+    implementations::resources::pools::{ProverPoolResource, ReplicaPoolResource},
+    service::{ServiceContext, StopReceiver},
+    task::Task,
+    wiring_layer::{WiringError, WiringLayer},
+};
+
+const SCRAPE_INTERVAL: Duration = Duration::from_secs(60);
+
+pub struct HousekeeperLayer {
+    house_keeper_config: HouseKeeperConfig,
+    fri_prover_config: FriProverConfig,
+    fri_witness_generator_config: FriWitnessGeneratorConfig,
+    fri_prover_group_config: FriProverGroupConfig,
+    fri_proof_compressor_config: FriProofCompressorConfig,
+}
+
+impl HousekeeperLayer {
+    pub fn new(
+        house_keeper_config: HouseKeeperConfig,
```

Comment:
Note that in the core the name consists of two words: `house_keeper` and `HouseKeeper`. Meanwhile, in the framework, the different notation is used: `housekeeper` and `Housekeeper`.
Please change the framework so that in matches the core (including this module name).